src/Perpetuum/Zones/Effects/ZoneEffects/ZoneEffectHandler.cs

```diff
@@ -1,22 +1,86 @@
-﻿using Perpetuum.Players;
+﻿using Perpetuum.Log;
+using Perpetuum.Players;
+using Perpetuum.Units;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace Perpetuum.Zones.Effects.ZoneEffects
 {
     public class ZoneEffectHandler : IZoneEffectHandler
     {
-        private readonly IZoneEffectRepository _zoneEffectRepository;
-        public ZoneEffectHandler(IZoneEffectRepository zoneEffectRepository)
+        private const byte _ = 0;
```

Comment:
If you *really* want to do this instead of passing 0, I suggest extended the `byte` object with a `byte.Zero` method that has the following attribute: `[MethodImpl(MethodImplOptions.AggressiveInlining)]`

Reply:
Well I just don't want to expend extra cpu cycles on adding what is a no-op value to abuse ConcurrentDict as a ConcurrentSet (why doesn't C# provide this?!).  
Passing 0 I still need to cast to byte, so wasn't sure how that would work out, but byte.MinValue does the trick, no?

Reply:
Should do, the option I gave was so that we knew it would be aggresively inlined by the compiler (essentially it should just inline the 0, removing the method call entirely). `MinValue` *should* work but I haven't checked to see if it would be inline or not.